community/modules/scheduler/pbspro-client/README.md

```diff
@@ -0,0 +1,122 @@
+## Description
+
+This module provisions 1 or more PBS Client hosts to submit jobs to a PBS
+Professional cluster. The following extra services are required:
+
+- an existing licensed PBS Professional Server; if provisioned in the cloud, we
+  recommend using the [pbspro-server module][pbspro-server].
+- a shared filesystem mounted across all PBS hosts to faciliate file transfers
+  for jobs and their stdin/stdout logs
```

Comment:
nit: for consistency, there should be a period at the end of this line like there is in the line above.

Reply:
Addressed on HEAD of PR